Remotion/Data/DomainObjects.UnitTests/Mapping/MappingReflectionIntegrationTests/MappingReflectionIntegrationTestBase.cs

```diff
@@ -48,6 +48,13 @@ public virtual void SetUp ()
       _typeDefinitions = typeDefinitions;
     }
 
+    [OneTimeSetUp]
+    public virtual void OneTimeSetUp ()
+    {
+      TestMappingConfiguration.Initialize();
+      StandardConfiguration.Initialize();
```

Comment:
I take it, the reflection tests fail if there's not `StandardConfiguration`? Which tests are affected?

Reply:
All of the `When_XXX` tests fail if they haven't set up the `StandardConfiguration` but they also do not inherit from `DatabaseTest` so the initialization needs to be done here.

Reply:
Loving the consistency :D